android/quest/src/androidTest/java/org/smartregister/fhircore/quest/ui/report/measure/components/ReportTypeSelectorScreenTest.kt

```diff
@@ -165,10 +165,10 @@ class ReportTypeSelectorScreenTest {
         dateRange = null,
         reportPeriodRange = mapOf(),
         onBackPressed = {},
-        onGenerateReport = {},
```

Comment:
Why was this removed?

Reply:
it fails while running other tests, that property must have removed